__tests__/features/fn_cancel_spec.js

```diff
@@ -95,13 +101,13 @@ describe('#fn_cancel', function () {
       });
 
       context('with :target', function () {
-        beforeEach(function () {
-          this.target = Helper.create('#target');
+        beforeEach(() => {
+          testContext.target = Helper.create('#target');
```

Comment:
💅 To avoid creating an extra variable and reset, we could just select this element on the expectation.